myapp/spec/models/tasks_spec.rb

```diff
@@ -65,10 +71,187 @@
       task = Task.new(
         title: 'more than 5 chars',
         description: 'more than 10 chars',
-        due: Time.zone.now - 60
+        due: Time.zone.now - 60,
+        status: 'pending'
       )
       
       expect(task).to_not be_valid
     end
+
+    it 'is invalid with no status specified' do 
+      task = Task.new(
+        title: 'title',
+        description: 'description',
+        due: Time.zone.now + 5
+      )
+
+      expect(task).to_not be_valid 
+    end
+  end
+
+  describe 'class method' do 
```

Comment:
`describe/context/it` would follow these rules.
- describe -> target method, class etc
- context -> test situation
- it -> expected results

example
```
  describe '.search_with_sort' do 
    context 'when query_status is empty' do 
      it 'sorted without status ' do 
```

Reply:
I removed `context` since it seems the situation is not that complex: 
a0aea73

Reply:
good!